app/dashboard/models.py

```diff
@@ -908,7 +913,7 @@ class Profile(SuperModel):
 
     """
 
-    user = models.OneToOneField(User, on_delete=models.CASCADE, null=True, blank=True)
+    user = models.OneToOneField(User, on_delete=models.SET(get_sentinel_user), null=True, blank=True)
```

Comment:
Can we just set this to: `on_delete=models.SET_NULL` - `deleted` user can only have one associated profile.